src/views/asset-edit-view/layout.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import { EditableAddress } from "../../components/edit-asset-address-form/editable-address";
+import { ReferenceAddress } from "../../components/edit-asset-address-form/reference-address";
+import { locale } from "../../services";
+
+import { Address, getAddressViaUprn } from "@mtfh/common/lib/api/address/v1";
+import { Asset, AssetAddress } from "@mtfh/common/lib/api/asset/v1";
+import { ErrorSummary, Link, StatusBox } from "@mtfh/common/lib/components";
+
+import "./styles.scss";
+
+export interface AssetEditLayoutProperties {
+  assetDetails: Asset;
+}
+
+export const AssetEditLayout = ({
+  assetDetails,
+}: AssetEditLayoutProperties): JSX.Element => {
+  const [currentAssetAddress, setCurrentAssetAddress] = useState<AssetAddress>(
+    assetDetails.assetAddress,
+  );
+  const [llpgAddress, setLlpgAddress] = useState<Address | null>(null);
+  const [showSuccess, setShowSuccess] = useState<boolean>(false);
+  const [showError, setShowError] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  useEffect(() => {
+    getAddressViaUprn(assetDetails.assetAddress.uprn).then((searchAddressResponse) => {
+      if (searchAddressResponse.addresses) {
+        setLlpgAddress(searchAddressResponse.addresses[0]);
+      }
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
No catch? Does setShowError get called anywhere?

Reply:
Added catch and made other error related changes